web-marketplace/src/hooks/useUpdateOrganizationProfile.ts

```diff
@@ -0,0 +1,265 @@
+import { useCallback } from 'react';
+import { useQueryClient } from '@tanstack/react-query';
+import { sanitizeDaoParams } from 'legacy-pages/CreateOrganization/hooks/useCreateDao/useCreateDao.utils';
+import {
+  DEFAULT_PROFILE_BG,
+  PROFILE_S3_PATH,
+} from 'legacy-pages/Dashboard/Dashboard.constants';
+import { useDashboardContext } from 'legacy-pages/Dashboard/Dashboard.context';
+import {
+  getExecuteActionsWasm,
+  getMsgExecuteContract,
+  WasmExecuteAction,
+} from 'utils/cosmwasm';
+
+import { useLedger } from 'ledger';
+import { useAuth } from 'lib/auth/auth';
+import { getAccountByAddrQueryKey } from 'lib/queries/react-query/registry-server/graphql/getAccountByAddrQuery/getAccountByAddrQuery.utils';
+import { getAccountByIdQueryKey } from 'lib/queries/react-query/registry-server/graphql/getAccountByIdQuery/getAccountByIdQuery.utils';
+
+import { orgRoles } from 'hooks/org-members/constants';
+import useMsgClient from 'hooks/useMsgClient';
+
+type UpdateOrganizationProfileParams = {
+  daoAddress: string;
+  rbamAddress: string;
+  organizationId?: string;
+  values: {
+    name: string;
+    description?: string;
+    profileImage?: string;
+    backgroundImage?: string;
+    websiteLink?: string;
+    twitterLink?: string;
+  };
```

Comment:
you could just use `EditProfileFormSchemaType` instead of manually typing the full type
this is more easily maintainable too: if we change the edit profile schema then this would be reflected here automatically too without having to change the `values` type manually...